### PR TITLE
Pbi 624 dataset thumbnails

### PR DESCRIPTION
**Pull request by edlu77**
#624

### CODE REVIEW

File: projects/ccf-eui/src/app/app.component.scss [hunk 1]
```diff
@@ -14,7 +14,6 @@
   height: 100%;
   overflow: hidden;
   font-size: 1rem;
-  color: black;
```
Comment (bherr2) [inline projects/ccf-eui/src/app/app.component.scss:17]: This was put here for the web components. What's the reason for removing this?

File: projects/ccf-database/src/lib/hubmap/hubmap-data.ts [hunk 2]
```diff
@@ -326,7 +326,10 @@ export class HuBMAPTissueBlock {
   }
 
   getDatasetThumbnail(dataset: JsonDict, assetsApi: string, serviceToken?: string): string | undefined {
-    if (dataset.group_uuid === '73bb26e4-ed43-11e8-8f19-0a7c1eab007a') { // TMC-Vanderbilt
+    if (dataset.thumbnail_file) {
+      const uuid = (dataset.thumbnail_file as JsonDict).file_uuid;
+      return `${uuid}/thumbnail.jpg` + (serviceToken ? `?token=${serviceToken}` : '');
```
Comment (bherr2) [inline projects/ccf-database/src/lib/hubmap/hubmap-data.ts:331]: Don't assume the filename is thumbnail.jpg. use `datasets.thumbnail_file?.filename` to get the filename.

File: projects/ccf-database/src/lib/hubmap/hubmap-data.ts [hunk 3]
```diff
@@ -326,7 +326,10 @@ export class HuBMAPTissueBlock {
...
+    } else if (dataset.group_uuid === '73bb26e4-ed43-11e8-8f19-0a7c1eab007a') { // TMC-Vanderbilt
```
Comment (bherr2) [inline projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: Did any datasets fall through to the other cases?

**Review by bherr2 — CHANGES_REQUESTED** [review]
Also revert package-lock.json changes

File: projects/ccf-database/src/lib/hubmap/hubmap-data.ts (hunk 3 above)
Comment (edlu77) [reply projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: Yes some of them don't have a thumbnail file.

Comment (bherr2) [reply projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: That's not the question

File: projects/ccf-eui/src/app/app.component.scss (hunk 1 above)
Comment (edlu77) [reply projects/ccf-eui/src/app/app.component.scss:17]: The text in dark mode is showing as the wrong color. How do we get around this?

Comment (bherr2) [reply projects/ccf-eui/src/app/app.component.scss:17]: In the EUI?

Comment (edlu77) [reply projects/ccf-eui/src/app/app.component.scss:17]: Yes

File: projects/ccf-database/src/lib/hubmap/hubmap-data.ts (hunk 3 above)
Comment (edlu77) [reply projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: What do you mean by other cases

Comment (bherr2) [reply projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: Were there any cases within that function where there was no thumbnail_file, but a thumbnail was found in the other two else clauses?

Comment (edlu77) [reply projects/ccf-database/src/lib/hubmap/hubmap-data.ts:332]: No it doesn't look like it.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**